### PR TITLE
Update run-localnet.md

### PR DESCRIPTION
**Pull request by paulmattei**

### CODE REVIEW

File: docs/run-localnet.md [hunk 1]
```diff
@@ -27,16 +27,18 @@ stop neard in case kuutamod is killed.
...
 
+It is expected that this guide is being followed on a OS you are familiar with.
+
 Install the nix package manager (as described [here](https://nix.dev/tutorials/install-nix)),
 and you will get all dependencies needed by running `nix develop` from the source directory
 of kuutamod
 
 ```console
 $ git clone https://github.com/kuutamolabs/kuutamod
-$ nix --extra-experimental-features "nix-command flakes" develop
```
Comment (Mic92) [inline docs/run-localnet.md:36]: Why is this dropped?

Comment (Mic92) [reply docs/run-localnet.md:36]: Any updates here?

Comment (paulmattei) [reply docs/run-localnet.md:36]: I added this by mistake as I was walking through the guide. This wasn't needed, so doc was updated, but it turned out that the person was on a nixos machine, that's why. This did need to be reverted. Done.

**Comment by paulmattei** [comment]
@Mic92 remind me what I need to do again when unit tests time out?

**Comment by Mic92** [comment]
Ignore for now.

**Comment by Mic92** [comment]
I think garnix might have build an ancient branch here anyway.

**Pull request merged**